libs/libcommon/src/libcommon/processing_graph.py

```diff
@@ -525,7 +525,7 @@ def parse_id(id: str) -> tuple[str, str, Optional[str], Optional[str], str]:
     },
     "split-descriptive-statistics": {
         "input_type": "split",
-        "triggered_by": "config-split-names",
+        "triggered_by": ["config-split-names", "config-parquet-metadata"],
```

Comment:
I added this because it requires parquet metadata, wdyt?

Reply:
OK! it explains why we have frequent `PreviousStepStillProcessingError`. This should fix it.

By the way, I think it's even:

```suggestion
        "triggered_by": "config-parquet-metadata",
```

because we don't require `config-split-names` to have been computed, we don't use it.

Reply:
surely test_processing_graph.py must be updated